core/Network/TLS/Parameters.hs

```diff
@@ -197,7 +197,8 @@ defaultSupported = Supported
     { supportedVersions       = [TLS12,TLS11,TLS10]
     , supportedCiphers        = []
     , supportedCompressions   = [nullCompression]
-    , supportedHashSignatures = [ (Struct.HashSHA512, SignatureRSA)
+    , supportedHashSignatures = [ (Struct.HashTLS13,  SignatureRSApssSHA256)
```

Comment:
Ordering is questionable, I think SHA-512 and SHA-384 should still have priority over SignatureRSApssSHA256.